roboflow/core/workspace.py

```diff
@@ -74,6 +75,34 @@ def project(self, project_name):
 
         return Project(self.__api_key, dataset_info, self.model_format)
 
+    def clip_compare(self, dir="", image_ext=".png", target_image=""):
+        """
+        @params:
+            dir: (str) = name reference to a directory of images for comparison
+            image_ext: (str) = file format for expected images (don't include the . before the file type name)
+            target_image: (str) = name reference for target image to compare individual images from directory against
+
+            returns: (dict) = a key:value mapping of image_name:comparison_score_to_target
+        """
+
+        assert isinstance(dir, str), "valid directory string required!"
```

Comment:
raise `ValueError` instead, `assert` is bad practice

Reply:
I have removed the `assert` statements. Since `AttributeError` would be the main cause for concern here I decided to remove error handling and allow `PIL` to do it for us.